src/mpl/src/timer/mpl_timer_clock_gettime.c

```diff
@@ -76,6 +77,9 @@ int MPL_wtick(double *wtick)
 
 int MPL_wtime_init(void)
 {
+    if (is_initialized)
+        goto fn_exit;
+
```

Comment:
Did you forget to set `is_initialized`?

Reply:
Also since there is no fancy error handling here, I would think a direct `return MPL_TIMER_SUCCSESS` and remove the label `fn_exit` to be cleaner.

Reply:
> Did you forget to set is_initialized?

D'uh.  Fixed now.

> Also since there is no fancy error handling here, I would think a direct return MPL_TIMER_SUCCSESS and remove the label fn_exit to be cleaner.

No, this is consistent with the rest of the MPICH code.